aspnetcore/blazor/components/integration.md

```diff
@@ -18,13 +18,210 @@ uid: blazor/components/integration
 
 This article explains Razor component integration scenarios for ASP.NET Core apps, including prerendering of Razor components on the server.
 
-Razor components can be integrated into Razor Pages and MVC apps. When the page or view is rendered, components can be prerendered at the same time. Prerendering can improve [Search Engine Optimization (SEO)](https://developer.mozilla.org/docs/Glossary/SEO) by rendering content for the initial HTTP response that search engines can use to calculate page rank.
+Razor components can be integrated into Razor Pages, MVC, and other types of ASP.NET Core apps with prerendering. Prerendering can improve [Search Engine Optimization (SEO)](https://developer.mozilla.org/docs/Glossary/SEO) by rendering content for the initial HTTP response that search engines can use to calculate page rank.
 
 Use the guidance in the following sections depending on the project's requirements:
 
+* Blazor support can be [added to an ASP.NET Core app](#add-blazor-support-to-an-aspnet-core-app).
 * For interactive components that aren't directly routable from user requests, see the [Use non-routable components in pages or views](#use-non-routable-components-in-pages-or-views) section. Follow this guidance when the app embeds components into existing pages and views with the [Component Tag Helper](xref:mvc/views/tag-helpers/builtin-th/component-tag-helper).
 * For interactive components that are directly routable from user requests, see the [Use routable components](#use-routable-components) section. Follow this guidance when visitors should be able to make an HTTP request in their browser for a component with an [`@page`](xref:mvc/views/razor#page) directive.
 
+## Add Blazor support to an ASP.NET Core app
```

Comment:
I think we should break this into three parts that build on each other: 

1. Adding static server rendering of Blazor components
    - I don't think this part should use a donor project - just show the minimal API changes and additions required
2. Enabling interactive server rendering
    - Again, no donor project. Just minimal changes. 
3. Enabling interactive WebAssembly rendering or Auto
    - Here I think a donor project makes sense for setting up the client project.